src/java/org/apache/nutch/tools/AbstractCommonCrawlFormat.java

```diff
@@ -23,6 +23,7 @@
 import java.net.URLEncoder;
 import java.net.UnknownHostException;
```

Comment:
Hi @thammegowda can you pleaae reformat this entire file to have 2 space indents here? I can review this PR once that is done. Thanks